mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -73,9 +73,8 @@ def __remote_init(self):
 
         # Remote DUT connection, flashing and reset...
         try:
-            self.__remote_connect(baudrate=self.baudrate)
-            self.__remote_flashing(self.image_path)
             self.__remote_disconnect()
```

Comment:
if check first if there was connection - if was then disconnect. this is neede to select right serialport configuration when connecting again..